ci_build_images/rhel.Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN --mount=type=secret,id=rhel_orgid,target=/run/secrets/rhel_orgid \
     "https://kojipkgs.fedoraproject.org/packages/Judy/1.0.5/28.fc36/$(arch)/Judy-devel-1.0.5-28.fc36.$(arch).rpm" \
     && dnf -y builddep mariadb-server \
     && dnf -y install \
-    ${extra} \
+    "${extra}" \
```

Comment:
because $extra under ubi9 contains two elements it cannot exist as a single unglob expanded argument.